test/Scaffolding/E2E_Test/Compiler/Resources/SharedViews/_ValidationScriptsPartial.cshtml

```diff
@@ -1,16 +1,16 @@
 ﻿<environment names="Development">
     <script src="~/lib/jquery-validation/dist/jquery.validate.js"></script>
-    <script src="~/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.js"></script>
+    <script src="~/lib/jquery-validation-unobtrusive/dist/jquery.validate.unobtrusive.js"></script>
 </environment>
 <environment names="Staging,Production">
-    <script src="https://ajax.aspnetcdn.com/ajax/jquery.validate/1.17.0/jquery.validate.min.js"
+    <script src="https://ajax.aspnetcdn.com/ajax/jquery.validate/1.19.3/jquery.validate.min.js"
```

Comment:
This is 1.21.0 https://www.npmjs.com/package/jquery-validation

Reply:
It's what's on the other templates and I think the identity UI has either 1.21.0 or 1.20.0. We need to update the integrity shas

Reply:
Version 1.19.3 is the latest version where that URL doesn't 404 (this file is used in E2E testing only).

@deepchoudhery, do you know what the process is like to get the latest version of `jquery-validation` on `https://ajax.aspnetcdn.com`?

Reply:
not quite no, these tests are quite out of date and most of them are skipped afaik. I would just match this file to the one shipping with Identity for now (bootstrap 5?). Will be writing a bunch of E2E for dotnet-scaffold soon.